src/ploneintranet/workspace/browser/tiles/sidebar.py

```diff
@@ -452,7 +454,7 @@ def _extract_attrs(self, catalog_results):
                 url=url,
                 creator=api.user.get(username=r['Creator']),
                 modified=r['modified'],
-                subject=r['Subject'],
+                subject=r.get('Subject', ()),
                 UID=r['UID'],
                 path=r.getPath()
             ))
```

Comment:
I'd rather change this code to read the variables that the search api already provides, rather than adding extra methods to the api.